sycl/source/detail/builtins_common.cpp

```diff
@@ -22,7 +22,7 @@
 namespace s = sycl;
 namespace d = s::detail;
 
-__SYCL_INLINE_NAMESPACE(cl) {
+// TODO:
```

Comment:
TODO what? (Also applies to sycl/source/detail/builtins_geometric.cpp, sycl/source/detail/builtins_helper.hpp, and sycl/source/detail/builtins_integer.cpp)